client/state/data-layer/wpcom/read/streams/index.js

```diff
@@ -193,6 +193,21 @@ const streamApis = {
 	following: {
 		path: () => '/read/following',
 		dateProperty: 'date',
+		query: ( extras ) => {
+			// Filter out undefined values from extras
+			const filteredExtras = Object.fromEntries(
+				Object.entries( extras ).filter( ( [ , value ] ) => value !== undefined )
+			);
```

Comment:
The `extras` parameter introduces `feed_id` and `page` with `undefined` values, breaking some existing unit tests. To remedy this, we filter out `undefined` parameters.